portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,109 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+
+  <body style="padding-top:18%;" id = "index" >
```

Comment:
would prefer moving this to .css file :)

Reply:
nit: remove whitespace for consistency `..." id="index">`

Reply:
Okay! moved to css file and cleaned up the whitespaces